### ISSUE TITLE
Unclear message when installing an image

### ISSUE THREAD

**Issue by vladimir-v-diaz** [uptane/obsolete-reference-implementation#57]
I sometimes come across the following message when installing an image on a primary client:

```Python
Successfully downloaded a trustworthy 'evil' image.
Submitting the Primary's manifest to the Director.
Submission of Vehicle Manifest complete.
>>> 127.0.0.1 - - [03/Apr/2017 12:53:17] "POST /RPC2 HTTP/1.1" 200 -
Treating requester as full-verification Secondary without a CAN interface because the C CAN interface is off or the ECU Serial ('22222') does not appear in the mapping of ECU Serials to CAN IDs.
```

It's not exactly clear to new users what this means, or if it's an error.

**Comment by awwad** [comment]
Yeah, I should probably kill that message.

This is from work toward use of C-based CAN code. If a particular ECU was associated in configuration with a CAN ID, and the CAN interface was available, code was supposed to try communicating across CAN. This is not used, so it should at least not spam. I'll update it.

**Comment by awwad** [comment]
Resolved by a3136cab96b77dbf77155b5c1c965ed74fecce58.
